File: C/EX18.cpp
#include <conio.h>
#include <stdlib.h>
#include <stdio.h>
#include <time.h> 
#include <string.h>
#include <math.h>
#include <locale.h>
#include <ctype.h>

int peso,retorno;
float altura,imc;
char sexo[10];
char base1[10] = "masculino";
int x=0;

int main()
{
	while(x==0)
	{
		printf("\nInsira o peso\n");
		scanf("%d",&peso);
		printf("\nInsira a altura em cm\n");
		scanf("%f",&altura);
		printf("\nInsira o sexo\n");
		scanf("%s",&sexo);
		
		imc=peso/(altura*altura);
		
		retorno = strcmp (sexo,base1);
		
		if(retorno == 0)
		{
			if(imc<20)
			{
				printf("\nVoce esta abaixo do peso\n");
			}
			else if(imc>=25)
			{
				printf("\nVoce esta acima do peso\n");
			}
			else
			{
				printf("\nVoce esta no peso ideal");
			}
		}
		if(retorno != 0)
		{
			if(imc<19)
			{
				printf("\nVoce esta abaixo do peso\n");
			}
			if(imc>=24)
			{
				printf("\nVoce esta acima do peso\n");
			}
			else
			{
				printf("\nVoce esta no peso ideal");
			}
		}
		
	
	}
	return 0;
	
}

File: Python/EX18.py
alt = float(input('Digite a sua altura em (m): '))
 
peso = float(input('Digite o seu peso (em Kg): '))

sexo = input('Digite o sexo (m - masculino / f - feminino): ')

imc = peso / (alt * alt)

if(sexo == 'm'):
		
	if(imc<20):
			
		print("Voce esta abaixo do peso")
			
	elif(imc>=25):
			
		prit("Voce esta acima do peso")
			
	else:
			
	    print("Voce esta no peso ideal")
			
elif(sexo == 'f'):
		
	if(imc<19):
			
		print("Voce esta abaixo do peso")
			
	elif(imc>=24):
			
		print("Voce esta acima do peso")
			
	else:
			
		print("Voce esta no peso ideal")
			
		